### ISSUE TITLE
Display tooltip for existed toolbar items

### ISSUE THREAD

**Issue by hauvubhtech** [niuware/mui-rte#259]
I want to add a custom tooltip for existed toolbar items, How can I do that? Currently It does not show tooltip on toolbar item hovering, so I found it hard for user to quickly understand the action of toolbar item will be

**Comment by lorstenoplo** [comment]
Yeah I exactly felt the same issue, if you agree I can create and submit a pr.

**Comment by islalobo** [comment]
Also agree a tooltip would be great to have for displaying the key command

**Comment by cpetrescu96** [comment]
`customControls={[

          {
            name: "title",
            icon: <Tooltip title="Title"><Title fontSize={'small'}/></Tooltip>,
            type: "block",
          },
          {
            name: "justifyLeft",
            icon: <Tooltip title="Justify text to Left"><FormatAlignLeft fontSize={'small'}/></Tooltip>,
            type: "block",
            blockWrapper: <JustifyLeft />
          },
        ]}`
        
I also encountered this issue, I can wrap a tooltip around the Icons of custom toolbar buttons but when I try to edit the default ones it doesn't work at all.
